curriculum/challenges/english/06-quality-assurance/quality-assurance-projects/metric-imperial-converter.md

```diff
@@ -8,8 +8,13 @@ forumTopicId: 301570
 ## Description
 <section id='description'>
 Build a full stack JavaScript app that is functionally similar to this: <a href="https://metric-imperial-converter.freecodecamp.rocks/" target="_blank">https://metric-imperial-converter.freecodecamp.rocks/</a>.
-Working on this project will involve you writing your code on Repl.it on our starter project. After completing this project you can copy your public Repl.it URL (to the homepage of your app) into this screen to test it! Optionally you may choose to write your project on another platform but it must be publicly visible for our testing.
-Start this project on Repl.it using <a href="https://repl.it/github/freeCodeCamp/boilerplate-project-metricimpconverter">this link</a> or clone <a href='https://github.com/freeCodeCamp/boilerplate-project-metricimpconverter/'>this repository</a> on GitHub! If you use Repl.it, remember to save the link to your project somewhere safe!
+Working on this project will involve you writing your code using one of the following methods:
```

Comment:
See above :arrow_up:

Reply:
Can you make the suggestion here on GitHub, so I can commit it or is that not possible?

Reply:
@RandellDawson Sorry, I would have, but GitHub does not allow suggestions across deleted sections. Unless, am I missing something?

Reply:
Done